### PR TITLE
Add vscode settings to gitignore.

### PR DESCRIPTION
**Pull request by nihohit**

### CODE REVIEW

**Comment by jaymell** [comment]
Personally I would rather we just a git-ignore entry for `.vscode`. That way we can have our own config and not have to worry about it overwriting / being overwritten by whatever's committed.

**Comment by nihohit** [comment]
done.

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,4 @@ build
 lib
 target
 .rust
+.vscode/settings.json
```
Comment (jaymell) [inline .gitignore:5]: I would just do `.vscode`

Comment (nihohit) [reply .gitignore]: great idea!

**Pull request merged**